recipes/jina-hubble-sdk/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "jina-hubble-sdk" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jina-hubble-sdk-{{ version }}.tar.gz
+  sha256: bab9b7b4d8c5965e099e07effd92cc0607b56eead0b4d26d6e5b473cb93daf8f
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - setuptools >=18.0
+    - wheel
+    - pip
+  run:
+    - python >=3.7
+    - requests ==2.27.1
+    - aiohttp ==3.8.1
```

Comment:
Do these really need to be pinned exactly to these versions? This usually eventually leads to problems resolving environments that install this package alongside others.

Reply:
we use `grayskull pypi --strict-conda-forge` jina-hubble-sdk`` generate the `meta.yaml`. and our repo ` jina-hubble-sdk`'s [requirements]( https://github.com/jina-ai/jina-hubble-sdk/blob/v0.2.0/requirements.txt#:~:text=requests%3D%3D2.27.1,aiohttp%3D%3D3.8.1)

Reply:
> Do these really need to be pinned exactly to these versions?

Yes, keep it consistent with the git version.